tests/ui-testing/pages/logsPage.js

```
@@ -367,9 +367,75 @@ export class LogsPage {
   
+    const rows = await this.page.locator('[data-test="logs-search-result-logs-table"] tbody:nth-of-type(2) tr').all();
+
+    for (let i = 2; i < rows.length; i++) {
+      const dateText = await rows[i].locator('td:first-child div div:nth-child(2) span span').textContent();
+      if (dateText) {
+        const date = new Date(dateText.trim());
+        const startDate = new Date(startDateStr);
+        const endDate = new Date(endDateStr);
+    
+        // Check if the parsed date is valid
+        if (isNaN(date.getTime())) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)